boards/arm/nrf52840_pca10090/board.c

```diff
@@ -218,7 +214,7 @@ static void reset_pin_wait_low(struct device *port, u32_t pin)
 
 	/* Wait until the pin is pulled low */
 	do {
```

Comment:
... and here do:
```
/* Wait until pin becomes active */
while (gpio_pin_get(port, pin) == 0) {
}
```
Other boards might include pins that are active-high, and using the logic level rather than physical level would make the code more consistent.

Reply:
I now see the original comment about using `_raw` to save a few bytes of flash.  I prefer clarity of using normal logic, but if I'm alone in that I won't insist.

Reply:
> Other boards might include pins that are active-high, and using the logic level rather than physical level would make the code more consistent.

This reset pin here is in fact active high. And the above loop waits until it becomes inactive. As the [accompanying comment](https://github.com/zephyrproject-rtos/zephyr/blob/7c9482e12d7a929c2486aa13d74b81ecbb8c31f7/boards/arm/nrf52840_pca10090/board.c#L275) says, this "lets the other side ensure that they are ready".
I'd prefer to left it as it is.